jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/navigation/Drawer.java

```diff
@@ -26,35 +26,35 @@ public class Drawer extends UIBaseElement<DrawerAssert> {
      * Gets lists within this drawer.
      *
      * @return lists included in this drawer as {@link List}
-     * @see MUIList
+     * @see MUISimpleList
      */
     @JDIAction("Get '{name}'s lists of items")
-    public List<MUIList> lists() {
+    public List<MUISimpleList> lists() {
         return core().finds(".MuiList-root").stream()
-                .map(list -> new MUIList().setCore(MUIList.class, list))
+                .map(list -> new MUISimpleList().setCore(MUISimpleList.class, list))
                 .collect(Collectors.toList());
     }
 
     /**
      * Gets the top (first) list of this drawer.
      *
-     * @return top list of this drawer as {@link MUIList}
-     * @see MUIList
+     * @return top list of this drawer as {@link MUISimpleList}
+     * @see MUISimpleList
```

Comment:
I think "@see MUISimpleList" is redundant here because there is already a reference to this class.